### PR TITLE
2.0 client config updates: no fedora28, yes el8

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: configs/foreman/2.0.yaml [hunk 1]
```diff
@@ -408,32 +409,33 @@
...
       build:
         - bash
         - bzip2
+        - centos-release
         - coreutils
         - cpio
         - diffutils
-        - fedora-release
         - findutils
+        - foreman-build
```
Comment (ekohl) [inline configs/foreman/2.0.yaml]: Do we include this in client repos?

File: configs/foreman/2.0.yaml [hunk 2]
```diff
@@ -408,32 +409,33 @@
...
     external_repos:
-      - fedora-28-updates
-      - fedora-28
+      - centos8-baseos
+      - centos8-appstream
     build_groups:
       build:
         - bash
         - bzip2
+        - centos-release
```
Comment (ekohl) [inline configs/foreman/2.0.yaml]: @pcreech has suggested in the past that we can use `redhat-release` which both `centos-release` and `fedora-release` provide. Might be good to consider but probably first in nightly.

File: configs/foreman/2.0.yaml (hunk 1 above)
Comment (evgeni) [reply configs/foreman/2.0.yaml]: I copied what we have in nightly. My best guess is no, but what do I know ;)

File: configs/foreman/2.0.yaml (hunk 2 above)
Comment (evgeni) [reply configs/foreman/2.0.yaml]: Agreed, mind making a card/redmine for that?

File: configs/foreman/2.0.yaml (hunk 1 above)
Comment (evgeni) [reply configs/foreman/2.0.yaml]: Would you like to block this until we find out, or postpone that?

Comment (ekohl) [reply configs/foreman/2.0.yaml]: Not a blocker I think

File: configs/foreman/2.0.yaml (hunk 2 above)
Comment (evgeni) [reply configs/foreman/2.0.yaml]: (done)

File: configs/foreman/2.0.yaml (hunk 1 above)
Comment (evgeni) [reply configs/foreman/2.0.yaml]: added a card to our backlog

**Pull request merged**